util/http.py

```diff
@@ -234,8 +234,8 @@ def _request_with_timeout(cls, url, make_request_with, *args, **kwargs):
 
         try:
             if verbose:
-                logging.info("Sending %s request to %s: kwargs %r",
-                             http_method, url, kwargs)
+                logging.info("Sending request to %s: args %r kwargs %r",
+                             url, args, kwargs)
```

Comment:
This is an unrelated problem I found while debugging. There's no variable `http_method` in this method -- it's one of the `args`.